src/webapp/src/client/app/wall/wall.html

```diff
@@ -1,10 +1,13 @@
 <div class="row body-container" has-permissions="['POST_BASIC']">
+    <div class="hidden-lg hidden-md">
```

Comment:
Just hiding doesn't disable DOM elements completely, they just not visible in the browser.
It means, that if some data is being loaded for these elements - it will be still loaded, even if not presented to the user.
And then it means, that there will be extra requests to the backend.

Please review and validate network traffic in the browser for extra requests. All Kudos related requests are quite heavy (unless not cached). So it's important to keep it optimized.

Reply:
I think wall-widget has the same problem, because it is always loaded even when it isn't shown. Should I look into it also?

Reply:
> I think wall-widget has the same problem, because it is always loaded even when it isn't shown. Should I look into it also?

Sure, nothing bad if it will be fixed. ;) But it can be considered as separate issue/pull-request (depends how big would be the change).